### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: docs/topic_guide/coordinates/carrington.rst [hunk 1]
```diff
@@ -122,7 +122,7 @@ Footnotes
 
 .. [#apparent] The term "apparent" (as opposed to "true") is used to indicate that the observer perceives the locations and orientations of objects different from how they truly are due to effects that include light travel time.
 .. [#subEarth] "Sub-Earth" is a special case of "sub-observer", where the observer is at Earth (specifically, Earth center).  That is, the sub-Earth point is the point on the Sun's surface "below" the center of the Earth along the Sun-Earth line.
-.. [#Carrington] Carrington (1863), *Observations of the Spots on the Sun*, p. 16 (`<https://archive.org/details/observationsofsp00carr/page/16/mode/1up>`__) and p.27 (`<https://archive.org/details/observationsofsp00carr/page/27/mode/1up>`__)
```
Comment (nabobalis) [inline docs/topic_guide/coordinates/carrington.rst:125]: RIP archive.

File: sunpy/net/hek/tests/test_hek.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def test_mixed_results_get():
                            attrs.hek.FRM.Name == 'SPoCA')
     assert isinstance(result, hek.hek.HEKTable)
     assert len(result) == 89
-    assert result[0]["SOL_standard"] == 'SOL2013-01-31T20:13:31L253C063'
+    assert result[0]["SOL_standard"].startswith("SOL2013-01-31T20:13:31")
```
Comment (nabobalis) [inline sunpy/net/hek/tests/test_hek.py:183]: This protects against the last set of numbers changing.

**Comment by wtbarnes** [comment]
Re: the paper references that are being fixed here, I just want to again advocate for #7078

File: pyproject.toml [hunk 3]
```diff
@@ -32,7 +32,7 @@ dependencies = [
   "numpy>=1.23.5",
   "packaging>=23.0",
   "parfive[ftp]>=2.0.0",
-  "pyerfa>=2.0",
+  "pyerfa>=2.0.0.1",
```
Comment (nabobalis) [inline pyproject.toml:35]: This version has a wheel for 310 at least.

File: docs/topic_guide/coordinates/carrington.rst (hunk 1 above)
Comment (nabobalis) [reply docs/topic_guide/coordinates/carrington.rst:125]: Does anyone have links I can use for this?

File: docs/topic_guide/coordinates/carrington.rst [hunk 4]
```diff
@@ -122,7 +122,7 @@ Footnotes
...
+.. [#Carrington] Carrington (1863), *Observations of the Spots on the Sun*, `p. 16 and p.27 <https://www.google.com/books/edition/_/GY1DAAAAcAAJ>`__
```
Comment (Cadair) [inline docs/topic_guide/coordinates/carrington.rst:125]: As @alasdairwilson [suggested](https://matrix.to/#/%23sunpy-development%3Aopenastronomy.org/%24172909985870123GnDis%3Amatrix.org?via=cadair.com&via=matrix.org&via=openastronomy.org):

```suggestion
.. [#Carrington] Carrington (1863), *Description of a Singular Appearance seen in the Sun on September 1, 1859*, `<https://adsabs.harvard.edu/full/1859MNRAS..20...13C>`__
```

?

**Review by Cadair — APPROVED** [review]

File: docs/topic_guide/coordinates/carrington.rst (hunk 4 above)
Comment (nabobalis) [reply docs/topic_guide/coordinates/carrington.rst:125]: The title is different and it only has 4 pages, the references to other pages doesn't add up anymore. 

But either way, I don't care about citing Carrington correctly

Comment (Cadair) [reply docs/topic_guide/coordinates/carrington.rst]: oh yeah this isn't the same thing.

**Comment by nabobalis** [comment]
Oh I forgot to patch the astropy devdeps failures.

**Comment by nabobalis** [comment]
Patched it blindly.

File: sunpy/net/dataretriever/sources/tests/test_goes_ud.py [hunk 5]
```diff
@@ -24,8 +24,20 @@ def LCClient():
     ("timerange", "url_start", "url_end"),
     [(Time('1995/06/03 1:00', '1995/06/05'),
       'https://umbra.nascom.nasa.gov/goes/fits/1995/go07950603.fits',
-      'https://umbra.nascom.nasa.gov/goes/fits/1995/go07950605.fits'),
-     (Time('2008/06/02 12:00', '2008/06/04'),
+      'https://umbra.nascom.nasa.gov/goes/fits/1995/go07950605.fits')])
+def test_get_url_for_time_range_old(LCClient, timerange, url_start, url_end):
+    # NASA likes to break its servers quite often.
```
Comment (wtbarnes) [inline sunpy/net/dataretriever/sources/tests/test_goes_ud.py:29]: 😆

File: sunpy/net/dataretriever/sources/tests/test_goes_ud.py [hunk 6]
```diff
@@ -58,11 +70,11 @@ def test_get_overlap_urls(LCClient, timerange, url_start, url_end):
 @pytest.mark.parametrize(("timerange", "url_start", "url_end"),
                          [(a.Time("2009/08/30 00:10", "2009/09/02"),
                            "https://www.ncei.noaa.gov/data/goes-space-environment-monitor/access/science/xrs/goes10/gxrs-l2-irrad_science/2009/08/sci_gxrs-l2-irrad_g10_d20090830_v0-0-0.nc",
-                           "https://www.ncei.noaa.gov/data/goes-space-environment-monitor/access/science/xrs/goes14/xrsf-l2-avg1m_science/2009/09/sci_xrsf-l2-avg1m_g14_d20090902_v1-0-0.nc")])
+                           "https://www.ncei.noaa.gov/data/goes-space-environment-monitor/access/science/xrs/goes10/xrsf-l2-avg1m_science/2009/09/sci_xrsf-l2-avg1m_g10_d20090902_v1-0-0.nc")])
```
[the remark this answers is not in the document]
Comment (wtbarnes) [reply sunpy/net/dataretriever/sources/tests/test_goes_ud.py:73]: Maybe just comment out the old line then and add a note to that effect?

File: sunpy/net/hek/tests/test_hek.py (hunk 2 above)
Comment (wtbarnes) [reply sunpy/net/hek/tests/test_hek.py:183]: Can you add a comment that says that? Otherwise someone might go back to expecting exact equality

**Review by wtbarnes — APPROVED** [review]

File: sunpy/net/hek/tests/test_hek.py (hunk 2 above)
Comment (nabobalis) [reply sunpy/net/hek/tests/test_hek.py:183]: Added

File: pyproject.toml (hunk 3 above)
Comment (nabobalis) [reply pyproject.toml:35]: Oh no.

**Comment by nabobalis** [comment]
Fixed by adding comments.

**Pull request merged**